pkg/policy/mapstate.go

```diff
@@ -392,6 +449,23 @@ func (keys MapState) denyPreferredInsertWithChanges(newKey Key, newEntry MapStat
 	keys.redirectPreferredInsert(newKey, newEntry, adds, deletes)
 }
 
+func netsContainAny(a, b []*net.IPNet) bool {
```

Comment:
```suggestion
// netsContainAny checks whether any IP range in 'a' is a superset of any IP range in 'b'.
func netsContainAny(a, b []*net.IPNet) bool {
```

(This is probably also a good candidate to sit in `pkg/ip` as a generic IP range comparison function, in case we need it again in future)